scipy/stats/stats.py

```diff
@@ -6213,6 +6224,10 @@ def chisquare(f_obs, f_exp=None, ddof=0, axis=0):
     category are too small.  A typical rule is that all of the observed
     and expected frequencies should be at least 5.
 
+    Also, the sum of the observed and expected frequencies must be the same
+    for the test to be valid; `chisquare` raises an error if the sums do not
+    agree within a relative tolerance of ``1e-8``.
```

Comment:
```suggestion
    agree within a relative tolerance of ``1e-5``.
```
1e-8 didn't pass tests. Another option would be to better normalize the test that failed, but what should this be?

Reply:
Which test failed with 1e-8?

Reply:
`test_power_divergence_against_cressie_read_data`

Reply:
The numbers in the paper from which this data was taken was reported to just 5 significant digits (from an underlying calculation of about 6 significant digits).  I looked into the source of the data, which is the text "Analysis of Qualitative Data: Volume 1 Introductory Topics" by Shelby Haberman (any relation?).  Here's how we can compute the expected frequencies directly:

```
# From Shelby J. Haberman, Analysis of Qualitative Data: Volume 1 Introductory
#      Topics, Academic Press, New York, USA (1978).
obs = np.array([15, 11, 14, 17, 5, 11, 10, 4, 8, 10, 7, 9, 11, 3, 6, 1, 1, 4])
beta = -0.083769  # Haberman (1978), p. 15
i = np.arange(1, len(obs) + 1)
alpha = np.log(obs.sum() / np.exp(beta*i).sum())
expected_counts = np.exp(alpha + beta*i)
```
This gives:
```
In [2]: expected_counts                                                           
Out[2]: 
array([15.17105918, 13.95196853, 12.83083954, 11.79980035, 10.85161169,
        9.97961599,  9.17769067,  8.44020512,  7.76198121,  7.13825688,
        6.56465275,  6.03714134,  5.55201881,  5.10587895,  4.69558926,
        4.31826893,  3.97126869,  3.65215211])

In [3]: expected_counts.sum()                                                     
Out[3]: 147.0
```
You can incorporate this into the current PR however you like, or if you prefer, I can submit a PR to update the test.
Whichever way we do it, I think it is worthwhile, because a relative tolerance of 1e-8 is seems preferable to 1e-5.

Reply:
Incorporated.